subsys/bluetooth/shell/bt.c

```diff
@@ -1206,26 +1238,6 @@ static int cmd_adv_select(const struct shell *shell, size_t argc, char *argv[])
 	return -ENOEXEC;
 }
 
-static int cmd_adv_oob(const struct shell *shell, size_t argc, char *argv[])
```

Comment:
This is outside of #if defined(CONFIG_BT_EXT_ADV) now, so will fail when this is disabled.
Need to wrap this #if defined(CONFIG_BT_EXT_ADV) again.